commands/bootstrap/inputs.go

```diff
@@ -596,7 +613,47 @@ func initializeInputs(inputFilePath, resourcesPath string, configuration config.
 		}
 	}
 
+	if inputsPath == "" {
+
+		bSlice, err := yaml.Marshal(inputValues)
```

Comment:
Errors should not be silently ignored
See https://github.com/golang/go/wiki/CodeReviewComments#handle-errors